app/src/main/java/com/sunasterisk/music_72/screen/MainActivity.kt

```diff
@@ -17,10 +18,7 @@ class MainActivity : AppCompatActivity(), BottomNavigationView.OnNavigationItemS
     }
 
     private fun initView() {
-        supportFragmentManager
-            .beginTransaction()
-            .add(R.id.container, HomeFragment.newInstance())
-            .commit()
+        addFragmentToActivity(supportFragmentManager, HomeFragment.newInstance(), R.id.container)
```

Comment:
viết theo kiểu extension đi Đạt 
```suggestion
        supportFragmentManager.addFragmentToActivity(HomeFragment.newInstance(), R.id.container)
```